definitions/scenarios/satellite_upgrade.rb

```diff
@@ -74,6 +74,7 @@ def compose
       add_steps(
         Procedures::MaintenanceMode::EnableMaintenanceMode,
         Procedures::Crond::Stop,
+        Procedures::Timer::Stop,
```

Comment:
In theory this shouldn't be needed for the current upgrade, but if there are no timers then I assume this will be a noop.

Reply:
Correct.